_layouts/post.html

```diff
@@ -23,7 +23,6 @@
 {% endif %}
 
 <div class="my-article-title">{{ page.title }}</div>
-<hr/>
```

Comment:
I kinda liked the additional visual separation of level-1 heading and content. Can we keep this? Or alternatively add extra margin-bottom to the CSS for h1?

Reply:
Sure, either solution is fine by me. But we should do it for both guide and post, not just one of them as it currently is.